Big-Spender/readme.md

```diff
@@ -93,6 +104,9 @@ CREATE YOUR QUERY HERE
 
 ```sql
 INSERT YOUR QUERY HERE
+
+select suppliers.supplier, sum(spends.amount) from suppliers join spends on suppliers.id = spends.supplier_id                                        
+group by suppliers.supplier;
```

Comment:
nice job! Maybe we could use an alias for sum(spends.amount) to make it look a little cleaner in the results?